guide/02-api-overview/release_notes_210.ipynb

```diff
@@ -0,0 +1,897 @@
+    "  * Fixes issue where certain proxies blocked OAuth authentication\n",
+    "  * Fixes issue where server cannot be properly accessed with Windows Authentication on single-machine deployments\n",
+    "  * Fixes [**Public Repo Issue #1349**](https://github.com/Esri/arcgis-python-api/issues/1349) where connecting returns `InsecureRequestWarning` when setting `use_gen_token` and `verify_cert` to `True`\n",
+    "  * [`hosting_servers`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.GIS.hosting_servers)\n",
+    "    * Fixes issue where property was not honoring proxy methods\n",
+    "  * Fixes issue with connecting to servers protected by LDAP \n",
+    "* [`Item`](/python/api-reference/arcgis.gis.toc.html#item)\n",
+    "  * [`metadata`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.Item.metadata)\n",
+    "    * Fixes [**Public Repo Issue #611**](https://github.com/Esri/arcgis-python-api/issues/611) where assignment fails with strings longer than 32767 characters\n",
+    "  * [`usage()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.Item.usage)\n",
+    "    * Fixes issue where dates were transformed incorrectly if `as_df` argument was `True`\n",
+    "  * [`update()`](/python/api-reference/arcgis.gis.toc/#arcgis.gis.Item.update)\n",
+    "    * Fixes failure to update `item` if it resides in a folder\n",
+    "    * Removes `commentsEnabled` parameter from table as no longer supported on a per `item` basis\n",
+    "* [`ContentManager`](/python/api-reference/arcgis.gis.toc.html#contentmanager)\n",
+    "  * [`clone_items()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ContentManager.clone_items) \n",
+    "    * BUG-000147274 Fixes issue with failing to clone item metadata\n",
+    "    * BUG-000149255 Fixes `KeyError` when cloning Solutions apps\n",
+    "    * BUG-000152758 Fixes issue where layers in a group layer were not being cloned\n",
+    "    * BUG-000153477 Fixes issue where method attempted to clone Living Atlas layers\n",
+    "  * [`advanced_search()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ContentManager.advanced_search)\n",
+    "    * Updates to parameter table formatting\n",
+    "  * [`search()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ContentManager.search)\n",
+    "    * BUG-000152801 Fixes issue where `categories` paramter was improperly formatted in sent request\n",
+    "* [`Group`](/python/api-reference/arcgis.gis.toc.html#group)\n",
+    "  * Fixes [**Public Repo Issue #1279**](https://github.com/Esri/arcgis-python-api/issues/1279) issue where not all properties were being returned when searching so provider value was always returning `None`\n",
+    "* [`ResourceManager`](/python/api-reference/arcgis.gis.toc.html#resourcemanager)\n",
+    "  * [`get()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ResourceManager.get)\n",
+    "    * Fixes return value formatting when `try_jason=False`\n",
```

Comment:
jason seems like a cool guy, but we should probably try_json

Reply:
HA! Very clever...I actually lol'd when I read this. Fixed